### ISSUE TITLE
Red dot notification mac

### ISSUE THREAD

**Issue by xinin** [getstation/desktop-app#299]
### Description
I would like to request the implementation of "red dot" notifications or the traditional "badge app icon" feature in the Station app. Currently, it enhances user experience by providing a quick visual cue about new activities or updates within the app. This can be particularly useful for users to stay informed without actively checking the app.

### Feature Details
- **Red Dot Notifications:**
  - This involves adding a small red dot indicator, typically placed on the app icon, to signify new messages, updates, or any relevant information.

- **Badge App Icon:**
  - The traditional "badge app icon" feature involves displaying a numeric badge on the app icon to indicate the number of unread notifications or messages.

### Why It's Beneficial
- **Enhanced User Engagement:**
  - Visual indicators like red dots or badge app icons can attract user attention, encouraging them to check the app for new content or updates.

- **User Convenience:**
  - Users can quickly assess whether there are new activities within the app without having to open it, providing a more efficient and streamlined experience.

### Additional Context
<img width="70" alt="Captura de pantalla 2023-12-21 a las 12 03 28" src="https://github.com/getstation/desktop-app/assets/146713179/b45b8ff2-9ba2-4fa2-a35f-797358f97ede">

### Expected Behavior
I expect to see either "red dot" notifications or the option to enable the "badge app icon" in the Station app settings.

### Environment Details
- **Station App Version:** [Specify the version number, if applicable]
- **Platform:** [Specify the operating system, e.g., Windows, macOS, Linux]

Thank you for considering this feature request. I believe that implementing these visual indicators will significantly improve the user experience within the Station app.

Best regards,
Diego

**Comment by samufi** [comment]
I would like to second this feature request also for Windows.
More importantly, I would like to see such a visual cue on the app symbols within the application.

**Comment by josephhyatt** [comment]
this is also requested in the windows desktop app.. specifically for microsoft teams for me.

**Comment by MaZZly** [comment]
Yep, the red notification dot for individual apps would be really useful.. The thing I miss most since moving over from Ferdium.

Also, I noticed when marking off notifications in the "notifications menu", it still shows a red dot for that one, although there are no remaining items...

**Comment by angelmixu** [comment]
Hi! I was just starting to use Station and I'm really liking it, I was missing this badge/red dot notification (NixOS user here).

I tended to put whatsapp, telegram, gmail, etc. into pinned tabs in my browser, and gmail uses a little badge number through a plugin, while whatsapp and telegram for example, change the tab's icon, so I'm quite used to look at those cues to go to the proper webpage.

How's the state of it?
Let me know if I can help :)
